x-pack/plugins/lens/public/visualizations/tagcloud/suggestions.ts

```diff
@@ -42,8 +42,7 @@ export function getSuggestions({
       return {
         previewIcon: IconChartTagcloud,
         title: TAGCLOUD_LABEL,
-        hide: true, // hide suggestions while in tech preview
-        score: 0.1,
+        score: bucket.operation.dataType === 'string' ? 0.4 : 0.2,
```

Comment:
tagcloud makes more sense for string data than for numeric one, so I think it makes sense to differentiate it with the suggestions we show.